recipes/_lumberjack_secrets.rb

```diff
@@ -47,20 +47,36 @@
   Chef::Log.warn('Unable to complete lumberjack keypair configuration')
 end
 
-# if we had overrode basedir value, we'd need to use the new value here too
-file node['logstash_forwarder']['config']['network']['ssl key'] do
+# logstash secrets
+file node['lumberjack']['ssl_key_path'] do
   content node.run_state['lumberjack_decoded_key']
-  owner node['logstash_forwarder']['user']
-  group node['logstash_forwarder']['group']
+  owner node['lumberjack']['user']
```

Comment:
Should we name these lumberjack when it's the protocol vs. the software package's name?